activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Fix various ActiveSupport Range extensions when e.g. passed Range arguments, so they
+    work with beginless (startless) and endless ranges. Builds on work by Allen Hsu.
```

Comment:
Since there are just three methods, I think you can mention them specifically. eg. ```Fix `Range#===`, `Range#include?`, and `Range#cover?` to work with  beginless (startless) and endless range parameters.```

Reply:
👍

Reply:
`CHANGELOG.md` changes made & commits squashed.